### ISSUE TITLE
Invalid Path Export in setenv.sh

### ISSUE THREAD

**Issue by sDos280** [reapyr-dev/reapyr#8]
It seems that while running the `setenv.sh` in a directory one of its "parent" directories has a space in its name it will break the whole export part...
for example (in my case), if I will run the `setenv.sh` in a directory like "`/Coding Projects/reapyr/src/modules/quickstart`" it will break the whole export part.
the generated  `_setvars.sh`:
```bash
export PYTHONPATH="/home/dor283/Coding Projects/reapyr/src/modules/cpython/lib:/home/dor283/Coding Projects/reapyr/deps/shedskin"
export PATH=$PATH:/home/dor283/Coding Projects/reapyr/src/modules/quickstart
export REAPYR_SDK_ROOT=/home/dor283/Coding Projects/reapyr
bash
```

one fix that I can think of is just to wrap the stuff after the `export ...=` in `""`

**Comment by sDos280** [comment]
it seems that #1 is also related to that...

**Comment by sDos280** [comment]
closed since that is just the problem with #1 ...
